PencilCase/PencilCase/scripts/Models/Product.ts

```diff
@@ -0,0 +1,51 @@
+﻿import * as Utils from '../Utils';
+
+export class Product {
+    public Id: string = null;
+    public Name: string = '';
+    public Description: string = '';
+    public RetailPrice: number = 0;
+    public RetailUnit: string = null;
+    public WholesalePrice: number = 0;
+    public WholesaleUnit: string = null;
+    public ImportWholesalePrice: number = 0;
+    public ImportRetailPrice: number = 0;    
+    public Times: number = 0;
+    public Inventory: number = 0;
+    public Image: any = null;
+    public CreatedDate: Date;
+    public ModifiedDate: Date;
+
+    public RetailUnitName: string;
```

Comment:
Add these 2 fields to represent the name of unit since the actually value in database is a guid.